### ISSUE TITLE
"Cannot add extension with name 'checkerFramework'" in 0.5.6

### ISSUE THREAD

**Issue by mernst** [kelloggm/checkerframework-gradle-plugin#116]
When upgrading from 0.5.5 to 0.5.6, I get errors of the following form:

```
An exception occurred applying plugin request [id: 'java']
> Failed to apply plugin [id 'org.gradle.java']
   > Cannot add extension with name 'checkerFramework', as there is an extension already registered with that name.
```

Here are two projects with the error:

 * [error](https://app.circleci.com/pipelines/github/plume-lib/require-javadoc/120/workflows/6a0abac8-9056-44a6-81a7-e3028373d8c0/jobs/120) in [require-javadoc](https://github.com/plume-lib/require-javadoc/pull/52)
 * [error](https://dev.azure.com/mernstrandoop/randoop/_build/results?buildId=178&view=logs&jobId=174aab74-3b97-5c70-c4a5-0c95336847a1&j=174aab74-3b97-5c70-c4a5-0c95336847a1&t=0b3a3da1-3394-5f2c-7f2e-671e25bcc870) in [randoop](https://github.com/randoop/randoop)

Am I using the plugin incorrectly, or is there a bug in the plugin?

Thanks!

**Comment by mernst** [comment]
@kelloggm @9999years

**Comment by vlsi** [comment]
A side note: do you think it might be interesting to add a CI job to `checkerframework-gradle-plugin` so it validates PRs to `checkerframework-gradle-plugin` do not break a selected set of another projects? (e.g. `require-javadoc`, `randoop`, etc)

The flow could be:

1. Build `checkerframework-gradle-plugin` (e.g. with `1.0.0-dev-testing` version)
2. Checkout `require-javadoc`, and build it with `1.0.0-dev-testing` gradle plugin. It could be either `-Pcheckerframework-gradle-plugin` or it could be a special `shell` file inside `require-javadoc` project.

**Comment by 9999years** [comment]
Fix in #115. Sorry for the trouble!

I think validating releases against other projects wouldn't be a bad idea, but it's likely out of the scope of my changes.

**Comment by vlsi** [comment]
Of course, extra validation is out of scope for #115 :)

**Comment by mernst** [comment]
Fixed by #115.  Thanks!
